falcon/response.py

```diff
@@ -136,6 +136,20 @@ def set_stream(self, stream, stream_len):
         self.stream = stream
         self.stream_len = stream_len
 
+    def get_header(self, name):
+        """Check if a header is set on the response.
```

Comment:
Not just checking, but also returns the value? Perhaps reword to: "Get the value of a header set on..."